modules/launcher/src/main/java/org/mule/runtime/module/launcher/DefaultMuleContainer.java

```diff
@@ -415,29 +426,12 @@ public static String getStartupPropertiesFile() {
   }
 
   public static void setStartupPropertiesFile(String startupPropertiesFile) {
```

Comment:
this is only being called in the `init` method apparently... does this and the field have to be `static`?